tools/flang1/flang1exe/fpp.c

```diff
@@ -376,6 +376,20 @@ skipbl(char *tokval, int flag)
   return toktyp;
 }
 
+static void fpp_display_macros()
```

Comment:
I am assuming accpp.c is the default pre-processor and hence by default only changes in accpp.c is only tested. Can you add tests for fpp.c also by appropriately switching on flags?

Reply:
tests for fpp module also added in dm_option.f90
for example `! Check amd64-linux architecture but from FPP module
! RUN: %flang -Hx,123,0x4000000 -dM -E -target x86_64-unknown-linux-gnu %s | FileCheck %s \
! RUN: --check-prefixes=CHECK-MACROS-AMD64,CHECK-MACROS-LINUX,CHECK-MACROS-COMMON-FPP`
-->fixed